conda_smithy/configure_feedstock.py

```diff
@@ -361,9 +361,6 @@ def render_appveyor(jinja_env, forge_config, forge_dir):
         for case in old_matrix:
             case = odict(case)
 
-            # Use Python 2.7 as a fallback when no Python version is set.
-            case["CONDA_PY"] = case.get("CONDA_PY", "27")
```

Comment:
Might leave this around though as we could use it to change the default Miniconda used for those cases.